recipes/arro3/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ arro3_module }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kylebarron/arro3/archive/refs/tags/py-v0.2.1.tar.gz
```

We need this to get the autoupdate bot.

```suggestion
  url: https://github.com/kylebarron/arro3/archive/refs/tags/py-v{{ version }}.tar.gz
```